src/containers/profile-page/components/desktop/ProfilePage.tsx

```diff
@@ -493,7 +491,6 @@ const ProfilePage = ({
         key={index}
         size='medium'
         id={playlist.playlist_id}
-        userId={playlist.playlist_owner_id}
```

Comment:
If you look on the playlist tab of a user's profile with badges, it's now missing badges